app/clients/sms/aws_pinpoint.py

```diff
@@ -83,13 +84,20 @@ def send_sms(
         reference,
         multi=True,
         sender=None,
+        created_at=datetime.utcnow(),
         **kwargs,
     ):
         aws_phone_number = self.origination_number if sender is None else sender
         recipient_number = str(to)
 
         try:
             start_time = monotonic()
+            # Log how long it spent in our system before we sent it
+            self.logger.info(
+                'Total time spent to send %s notification: %s seconds',
+                SMS_TYPE,
+                (datetime.utcnow() - created_at).total_seconds(),
+            )
```

Comment:
shouldn't we log this after we actually send the request on the next line?

Reply:
No, the intent here is to know how long it takes in our system, so we can track improvements/degradation. If the provider takes 60s to respond that's not on us.

Reply:
Agree, it should be above the try.

Reply:
[Done](https://github.com/department-of-veterans-affairs/notification-api/pull/2295/commits/7c8ff0267789fec73027f91b7b57422a1c4b07bf)